pkgs/build-support/build-pecl.nix

```
@@ -34,4 +34,11 @@ stdenv.mkDerivation (args // {
       internalDeps}
   '';
   checkPhase = "NO_INTERACTON=yes make test";
+
+  passthru = {
+    // This flag has been introduced for `nix-update` so that tool is able to
+    // work correctly with PHP extensions.
+    // See the corresponding PR: https://github.com/Mic92/nix-update/pull/123
+    isPhpExtension = true;
+  };
```

```suggestion
  passthru = passthru // {
    # This flag has been introduced for `nix-update` so that tool is able to
    # work correctly with PHP extensions.
    # See the corresponding PR: https://github.com/Mic92/nix-update/pull/123
    isPhpExtension = true;
  };
```

We probably do not want to override the whole `passthru` attr set but only add the attribute `isPhpExtension`. We should do a merge like it is done for the Ruby gems. Note that you will need to add something like `, passthru ? { }` L15.

Also, comments in Nix uses `#` :smile:

---

Fixed. Thanks!